pkg/jx/cmd/create_vault.go

```diff
@@ -127,23 +145,53 @@ func (o *CreateVaultOptions) createVaultGKE(vaultName string) error {
 		return errors.Wrap(err, "creating vault operator client")
 	}
 
-	return o.createVault(vaultOperatorClient, vaultName)
+	return o.createVault(vaultOperatorClient, vaultName, teamSettings.KubeProvider)
 }
 
 // DoCreateVault creates a vault in the existing namespace.
 // If the vault already exists, it will error
-func (o *CreateVaultOptions) createVault(vaultOperatorClient versioned.Interface, vaultName string) error {
-	kubeClient, _, err := o.KubeClientAndNamespace()
-	if err != nil {
-		return err
-	}
+func (o *CreateVaultOptions) createVault(vaultOperatorClient versioned.Interface, vaultName string, kubeProvider string) error {
 	// Checks if the vault already exists
 	found := kubevault.FindVault(vaultOperatorClient, vaultName, o.Namespace)
 	if found {
 		return fmt.Errorf("Vault with name '%s' already exists in namespace '%s'", vaultName, o.Namespace)
 	}
 
-	err = gke.Login("", true)
+	kubeClient, _, err := o.KubeClientAndNamespace()
+	if err != nil {
+		return err
+	}
+	clusterName, err := cluster.ShortName(o.Kube())
+	if err != nil {
+		return err
```

Comment:
Please wrap this error

Reply:
I don't think it's needed since ShortName returns the error wrapped